### ISSUE TITLE
Different from lin yu-ching' s gitlab repo?

### ISSUE THREAD

**Issue by sisrfeng** [Chang-Chia-Chi/TrackNet-Badminton-Tracking-tensorflow2#4]
Thank you for your nice work.
What is the main defference between your repo and this one?
https://nol.cs.nctu.edu.tw:234/open-source/TrackNetv2/tree/master/3_in_3_out
![image](https://user-images.githubusercontent.com/53520949/119922221-f66e5d80-bfa1-11eb-84b8-19ccd1ff09ed.png)
And a similar one:
https://gitlab.com/lukelin/tracknet_practice

**Comment by Chang-Chia-Chi** [comment]
The basic idea is identical but I modified model structure and use different loss function.
CNN backbone and architecture of original paper is VGG and DeconvNet , I use ResNet and U-Net.
Loss function in original paper is binary cross entropy and I use focal loss to help model to focus more on small object.

BTW, I think structure used in this repo will achieve higher accuracy but inference time will also be longer. It's a trade-off to use more complex model.

**Comment by sisrfeng** [comment]
Thx!
The loss function in the paper is named Weighted Binary Cross Entropy loss, but it' s  actully focal loss.
My analyze can be found in : https://zhuanlan.zhihu.com/p/375841897
